hermes-management/src/main/java/pl/allegro/tech/hermes/management/api/TopicsEndpoint.java

```diff
@@ -105,6 +105,26 @@ public TopicMetrics getMetrics(@PathParam("topicName") String qualifiedTopicName
         return topicService.getTopicMetrics(TopicName.fromQualifiedName(qualifiedTopicName));
     }
 
+    @POST
+    @Produces(APPLICATION_JSON)
+    @Path("/{topicName}/blacklist")
+    @RolesAllowed(Roles.ADMIN)
+    @ApiOperation(value = "Blacklist topic", httpMethod = HttpMethod.POST)
+    public Response blacklistTopic(@PathParam("topicName") String qualifiedTopicName) {
+        topicService.blacklistTopic(qualifiedTopicName);
```

Comment:
make it blacklistService <- separate blacklist from topics altogether (including endpoints, as mentioned before)